zebra-scan/src/service/scan_task/executor.rs

```diff
@@ -1,45 +1,54 @@
 //! The scan task executor
 
+use std::collections::HashMap;
+
+use color_eyre::eyre::Report;
 use futures::{stream::FuturesUnordered, FutureExt, StreamExt};
 use tokio::{
     sync::mpsc::{Receiver, Sender},
     task::JoinHandle,
 };
 use tracing::Instrument;
-use zebra_chain::BoxError;
+use zebra_chain::transaction;
 
 use super::scan::ScanRangeTaskBuilder;
 
 const EXECUTOR_BUFFER_SIZE: usize = 100;
 
-pub fn spawn_init() -> (
-    Sender<ScanRangeTaskBuilder>,
-    JoinHandle<Result<(), BoxError>>,
-) {
+pub fn spawn_init(
+    subscribed_keys_receiver: tokio::sync::watch::Receiver<
+        HashMap<String, std::sync::mpsc::Sender<transaction::Hash>>,
+    >,
+) -> (Sender<ScanRangeTaskBuilder>, JoinHandle<Result<(), Report>>) {
     // TODO: Use a bounded channel.
```

Comment:
```suggestion
    // TODO: Use a bounded channel.
```

It looks like already bounded.

Reply:
Removed the TODO in https://github.com/ZcashFoundation/zebra/pull/8253/commits/b6f5461389d5af6a4c7e427f6a866b75a177b5c2.